src/python/pants/goal/anonymous_telemetry.py

```diff
@@ -0,0 +1,184 @@
+# Copyright 2021 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import annotations
+
+import json
+import logging
+import re
+import uuid
+from typing import cast
+
+from humbug.consent import HumbugConsent  # type: ignore
+from humbug.report import Report, Reporter  # type: ignore
+
+from pants.engine.internals.scheduler import Workunit
+from pants.engine.rules import collect_rules, rule
+from pants.engine.streaming_workunit_handler import (
+    StreamingWorkunitContext,
+    WorkunitsCallback,
+    WorkunitsCallbackFactory,
+    WorkunitsCallbackFactoryRequest,
+)
+from pants.engine.unions import UnionRule
+from pants.option.subsystem import Subsystem
+
+logger = logging.getLogger(__name__)
+
+
+_bugout_access_token = "3ae76900-9a68-4a87-a127-7c9f179d7272"
```

Comment:
I see no way around exposing these in the client code. So we are theoretically open to being spammed. But we might want to obfuscate a little somehow?  @zomglings, any thoughts or best practices here?

Reply:
Still mildly concerned about spamming, but I guess that is inevitable with telemetry. Even if we somehow obfuscated the hell out of this, someone could just run Pants itself in a spammy way. So I guess let's not worry about it until it happens.